shared_inc/wiki_functions.inc.php

```diff
@@ -12,18 +12,19 @@
 
 $needle = isset ($_REQUEST['needle']) ? $_REQUEST['needle'] : ""; 
 
-$lang = isset ($_REQUEST['lang']) ? htmlspecialchars($_REQUEST['lang']) : "";
-if($lang=="")
-{
-	$lang=$user_lang; 
-}
```

Comment:
@FlominatorTM, why is this code duplicated in `wikiblame.php`?

https://github.com/FlominatorTM/wikiblame/blob/3415d8033f72310fed1f77ff38b04837ccb196aa/wikiblame.php#L269-L272

I could change that the same way I did here, but I wonder if the duplication is not intentional.